src/static_julia.jl

```diff
@@ -122,33 +122,38 @@ function julia_compile(
         math_mode, depwarn
     )
 
-    shared && build_shared(s_file, joinpath(tmp_dir, o_file), verbose)
+    shared && build_shared(s_file, joinpath(tmp_dir, o_file), optimize, debug, verbose)
 
-    executable && compile_executable(s_file, e_file, cprog, verbose)
+    executable && compile_executable(s_file, e_file, cprog, optimize, debug, verbose)
```

Comment:
I don't like that I had to reorder these, since the command line options are ordered `-v -O0 -g2 -a -e`, but since verbose has a [default value](#diff-d47d0c2f9ab34ed28f73bdd671edd18cL148) (`verbose = false`), it had to go at the end.

I would be happy to do any of:
 1. leave it like this,
 2. add default values to `optimize` and `debug`, or
 3. remove the default value on `verbose`.

I'd vote for 3, since there's no one even using the function anywhere without specifying that value.

Reply:
I'm fine with 3!

Reply:
Great. Changed it! Thanks!